multiversx_sdk/network_providers/transaction_decoder.py

```diff
@@ -159,8 +159,11 @@ def get_nft_transfer_metadata(self, metadata: TransactionMetadata) -> Optional[T
         result.transfers = []
 
         if len(args) > 4:
-            result.function_name = self.hex_to_string(args[4])
-            result.function_args = args[5:]
+            if receiver.is_smart_contract():
```

Comment:
Here, `receiver` is the actual receiver of the tokens, not `tx.receiver`, correct?

Reply:
yep, the actual receiver